src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projectcreation/viewmodel/ProjectCreationViewModel.kt

```diff
@@ -9,17 +16,58 @@ class ProjectCreationViewModel : ItemViewModel<ProjectCreationModel>(ProjectCrea
     var targetLanguage = bind(ProjectCreationModel::targetLanguageProperty, true)
 
 
-    var resource = bind(ProjectCreationModel::resourceSelected, true)
+    var selectedResourceProperty = bind(ProjectCreationModel::selectedResource, true)
+    var selectedAnthologyProperty = bind(ProjectCreationModel::selectedAnthology, true)
+    val selectedBookProperty = bind(ProjectCreationModel::selectedBook, true)
+
+
+    val languagesListProperty= bind(ProjectCreationModel::languages)
+    var languageList: ObservableList<Language> = FXCollections.observableArrayList()
+
+    val resourceListProperty = bind(ProjectCreationModel::resources)
+    var resourceList : ObservableList<Collection> = FXCollections.observableArrayList()
+
+    val allProperty = bind(ProjectCreationModel::all)
+    var allList: ObservableList<Collection> = FXCollections.observableArrayList()
+
+    val anthologyListProperty = bind(ProjectCreationModel::anthologyList, true)
+    val anthologyList = FXCollections.observableArrayList<Collection>()
+
+    val bookListProperty = bind(ProjectCreationModel::bookList, true)
+    val bookList = FXCollections.observableArrayList<Collection>()
 
     val projectsProperty = bind(ProjectCreationModel::projectProperty)
+    val allPagesComplete = SimpleBooleanProperty(false)
+
+    init {
+
+        languagesListProperty.value.map {
+            languageList.setAll(it)
+        }.subscribe()
```

Comment:
It might just be a style preference, but the subscribe/assign could be in the model. Then the view model can just expose the observable list from the model. 

Also, since observable lists are observable by default, you can skip wrapping them in properties. So if we want to expose the languages observable list from the model to the view, the view model could just have a member like `val languages = model.languages`.